### PR TITLE
[Docs] Ability to use CNI chaining

### PR DESCRIPTION
**Pull request by amitmavgupta**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ For next steps on leveraging Network Policy, we encourage you to explore:
 
 ## Additional Cilium Features
 
-Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  Hubble observability, DNS-aware and HTTP-Aware Network Policy, Multi-cluster Routing, Transparent Encryption, and Advanced Load-balancing.
+Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:
```
Comment (chrisdoherty4) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]:
```suggestion
Some advanced features of Cilium are not enabled as part of EKS Anywhere, including:
```

Comment (chrisdoherty4) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]:
```suggestion
Some advanced features of Cilium are not enabled as part of EKS Anywhere, including:
```

File: docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ For more information on CNI customization see [Use a custom CNI]({{< ref "../../
 
 {{% alert title="Note" color="primary" %}}
 When replacing EKS Anywhere Cilium with a custom CNI, it is your responsibility to manage the custom CNI, including version upgrades and support.
+Operators can also leverage the CNI chaining feature from Isovalent where in both Cilium as the CNI and another CNI can work in a chain mode (https://docs.cilium.io/en/v1.13/installation/cni-chaining/)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md:14]:
```suggestion
Operators can also leverage the CNI chaining feature from Cilium where both Cilium and another CNI work in chain mode (https://docs.cilium.io/en/v1.13/installation/cni-chaining/)
```

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md (hunk 1 above)
Comment (raykrueger) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]: Is this the list of Open Source features that are not enabled or Enterprise features?

Comment (amitmavgupta) [reply docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]: @raykrueger The default eks-a image has a very basic set of features as illustrated on that page. That's right these are EE features. The way we are going with this is a comparison amongst what is there in the embedded version of EKS-A for Cilium vs. what is there in Isovalent Enterprise for Cilium.

@danwent

File: docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ For more information on CNI customization see [Use a custom CNI]({{< ref "../../
 
 {{% alert title="Note" color="primary" %}}
 When replacing EKS Anywhere Cilium with a custom CNI, it is your responsibility to manage the custom CNI, including version upgrades and support.
+Operators can also leverage the CNI chaining feature from Cilium where both Cilium and another CNI work in chain mode (https://docs.cilium.io/en/v1.13/installation/cni-chaining/)
```
Comment (chrisnegus) [inline docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md:14]:
```suggestion
Operators can also leverage the [CNI chaining]((https://docs.cilium.io/en/v1.13/installation/cni-chaining/) feature from Cilium where both Cilium and another CNI work in chain mode.
```

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md [hunk 4]
```diff
@@ -126,6 +126,12 @@ For next steps on leveraging Network Policy, we encourage you to explore:
...
 ## Additional Cilium Features
 
-Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  Hubble observability, DNS-aware and HTTP-Aware Network Policy, Multi-cluster Routing, Transparent Encryption, and Advanced Load-balancing.
+Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  
+
+* Hubble observability (https://docs.isovalent.com/operations-guide/features/network-visibility/index.html)
+* DNS-aware and HTTP-Aware Network Policy (https://docs.isovalent.com/quick-start/policy_lifecycle.html)
+* Multi-cluster Routing (https://docs.isovalent.com/operations-guide/features/cluster-mesh/index.html)
+* Transparent Encryption (https://docs.cilium.io/en/v1.13/security/network/encryption/)
+* Advanced Load-balancing (https://docs.isovalent.com/operations-guide/features/cilium-standalone-gateway.html)
```
Comment (chrisnegus) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:135]:
```suggestion
* [Hubble observability](https://docs.isovalent.com/operations-guide/features/network-visibility/index.html)
* [DNS-aware and HTTP-Aware Network Policy](https://docs.isovalent.com/quick-start/policy_lifecycle.html)
* [Multi-cluster Routing](https://docs.isovalent.com/operations-guide/features/cluster-mesh/index.html)
* [Transparent Encryption](https://docs.cilium.io/en/v1.13/security/network/encryption/)
* [Advanced Load-balancing](https://docs.isovalent.com/operations-guide/features/cilium-standalone-gateway.html)
```

**Review by chrisnegus — COMMENTED** [review]
A couple of suggestions for how links are done. Otherwise, I'd like a quick okay from @csplinter

File: docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ For more information on CNI customization see [Use a custom CNI]({{< ref "../../
 
 {{% alert title="Note" color="primary" %}}
 When replacing EKS Anywhere Cilium with a custom CNI, it is your responsibility to manage the custom CNI, including version upgrades and support.
+Operators can also leverage the [CNI chaining]((https://docs.cilium.io/en/v1.13/installation/cni-chaining/) feature from Cilium where both Cilium and another CNI work in chain mode.
```
Comment (csplinter) [inline docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md:14]: Does CNI chaining work with the EKS-A Cilium? We need those details before merging this.

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md (hunk 1 above)
Comment (csplinter) [reply docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]: This conflicts with #6060

File: docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md (hunk 5 above)
Comment (amitmavgupta) [reply docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md:14]: @csplinter yes its supported.

**Pull request closed without merging**